meetings/2020-02-10.md

```diff
@@ -0,0 +1,61 @@
+# **February 10, 2020**
+
+Agenda: [https://github.com/ampproject/meta-ac/issues/78](https://github.com/ampproject/meta-ac/issues/78)
+
+Facilitator: [@jorydotcom][jorydotcom]
+
+Scribe: 
+
+
+## **Attendees:**
+
+*   Guilherme Moser de Souza ([@mobtec][mobtec])
+*   Levi Durfee ([@levidurfee][levidurfee])
+*   Melanie Sumner ([@melsumner][melsumner])
+*   Maggie Wettergreen ([@mjwettergreen][mjwettergreen])
+*   Pablo Delgado ([@pdelgadorodriguez][pdelgadorodriguez])
+*   Senthil Padmanabhan ([@senthilp][senthilp])
+*   Ted Shuter ([@TedShuter][TedShuter])
+*   Joe Alicata ([@wirelessjoe][wirelessjoe])
+
+## **Excused:**
+
+*   Kenji Baheux ([@kenjibaheux][kenjibaheux])
+*   Sumantro Das ([@sumodas][sumodas])
+*   Tobie Langel ([@tobie][tobie])
+*   Ali Ghassemi ([@aghassemi][aghassemi])
+*   Elisa Budelli ([@elibud][elibud])
+
+## **Absent:**
+
+*   Charles Vazac ([@cvazac][cvazac])
+*   David Merrell ([@dymerrell][dymerrell])
+*   Melissa DePuydt [@msteffan][msteffan])
```

Comment:
Melissa is excused. She can't make this time.

Reply:
fixed! Thank you @tobie